docs/docs/code/nodejs/working-with-files/README.md

```diff
@@ -74,20 +74,364 @@ export default defineComponent({
 });
 ```
 
-## Download a file to `/tmp`
+### Download a file to `/tmp`
 
 [See this example](/code/nodejs/http-requests/#download-a-file-to-the-tmp-directory) to learn how to download a file to `/tmp`.
 
-## Upload a file from `/tmp`
+### Upload a file from `/tmp`
 
 [See this example](/code/nodejs/http-requests/#upload-a-file-from-the-tmp-directory) to learn how to upload a file from `/tmp` in an HTTP request.
 
-## Download a file, uploading it in another `multipart/form-data` request
+### Download a file, uploading it in another `multipart/form-data` request
 
 [This workflow](https://pipedream.com/@dylburger/download-file-then-upload-file-via-multipart-form-data-request-p_QPCx7p/edit) provides an example of how to download a file at a specified **Download URL**, uploading that file to an **Upload URL** as form data.
 
-## Download email attachments to `/tmp`, upload to Amazon S3
+### Download email attachments to `/tmp`, upload to Amazon S3
 
 [This workflow](https://pipedream.com/@dylan/upload-email-attachments-to-s3-p_V9CGAQ/edit) is triggered by incoming emails. When copied, you'll get a workflow-specific email address you can send any email to. This workflow takes any attachments included with inbound emails, saves them to `/tmp`, and uploads them to Amazon S3.
 
 You should also be aware of the [inbound payload limits](/limits/#email-triggers) associated with the email trigger.
+
+
+## File Stores
```

Comment:
@ctrlaltdylan I think we should make this its own section of the docs at `/docs/file-stores` like we have `/docs/data-stores`. Then I think we can reference the file store docs near the top of these Node / Python docs and keep some of the `$.files` code examples here too, but since the File Store itself has UI and everything I think it makes sense to promote at the top level.